indico/modules/events/layout/templates/layout_meeting_lecture.html

```diff
@@ -1,10 +1,24 @@
 {% extends 'layout/base.html' %}
-{% from 'forms/_form.html' import simple_form %}
+{% from 'forms/_form.html' import simple_form, form_rows %}
 
 {% block title %}
     {%- trans %}Layout{% endtrans -%}
 {% endblock %}
 
 {%- block content %}
-    {{ simple_form(form, save_reminder=true, back_button=false) }}
+    {% call simple_form(form, save_reminder=true, back_button=false) %}
+         <div>
```

Comment:
Cause in javascript I'm removing elements after the theme selector when changing the theme, so if its not in a div it will remove the Save button.